src/Design/FileContent.cpp

```diff
@@ -630,8 +630,8 @@ bool FileContent::diffTree(NodeId root, const FileContent* oFc, NodeId oroot,
     }
     std::string symb1;
     std::string symb2;
-    if (current1.m_name) symb1 = Name(id1);
-    if (current2.m_name) symb2 = oFc->Name(id2);
+    if (current1.m_name) symb1 = std::to_string(Name(id1));
```

Comment:
In this place, the symbol-ID that was coming back was interpreted as number and so assigned a single char to the `symb1`. Unclear what was meant, but at least I am translating it to the symbol-ID now.